CHANGELOG.md

```diff
@@ -3,52 +3,58 @@
+* `vcd_vapp_network` does not default `dns1` and `dns2` fields to 8.8.8.8 and 8.8.4.4 respectively
+  [GH-455,GH-468]
+* `vcd_org_vdc` can be created with Flex allocation in vCD 9.7 and later. Also two new fields added
+  for Flex - `elasticity`, `include_vm_memory_overhead` [GH-443]
 * `resource/vcd_org` and `datasource/vcd_org` include a section `vapp_lease` and a section
   `vapp_template_lease` to define lease related parameters of depending entities - [GH-432]
-* `resource/vcd_vapp_vm` and `datasource/vcd_vapp_vm` include a field `adapter_type` in `network`
-  defintion to specify NIC type - [GH-441]
+* `resource/vcd_vapp_vm` Internal disks in VM template can be edited by `override_template_disk`
+  field [GH-412]
 * `vcd_vapp_vm` `disk` has new attribute `size_in_mb` [GH-433]
+* `resource/vcd_vapp_vm` and `datasource/vcd_vapp_vm` get optional `network_dhcp_wait_seconds` field
+  to ensure `ip` is reported when `ip_allocation_mode=DHCP` is used [GH-436]
+* `resource/vcd_vapp_vm` and `datasource/vcd_vapp_vm` include a field `adapter_type` in `network`
+  definition to specify NIC type - [GH-441]
+* `resource/vcd_vapp_vm` and `datasource/vcd_vapp_vm` `customization` block supports all available
+  features [GH-462]
 * `datasource/*` - all data sources return an error when object is not found [GH-446]
 
 DEPRECATIONS:
-* `resource/vcd_vapp_vm` `network.name` deprecated automatic attaching of vApp Org network when `network.type=org` and it doesn't exist. Requires to create/attach vApp Org network with `vcd_vapp_org_network` before referencing it.   
+* `resource/vcd_vapp_vm` `network.name` deprecates automatic attachment of vApp Org network when
+  `network.type=org` and it is not attached with `vcd_vapp_org_network` before referencing it.
+  [GH-455]
+* `resource/vcd_vapp_vm` field `initscript` is now deprecated in favor of
+  `customization.0.initscript` to group all guest customization settings [GH-462]
 
 BUG FIXES:
 
-* fix `vcd_vapp_vm` resource read - independent disks where losing `bus_number` and `unit_number` values after refresh. [GH-433]
-* `datasource/vcd_nsxv_dhcp_relay` crashes if no DHCP relay settings are present in Edge Gateway [GH-446]
-* `vcd_vapp_vm` `disk` has new attribute `size_in_mb` [GH-433]
-* `resource/vcd_org` and `datasource/vcd_org` include a section `vapp_lease` and a section
-  `vapp_template_lease` to define lease related parameters of depending entities - [GH-432]
+* `resource/vcd_vapp_vm` read - independent disks where losing `bus_number` and `unit_number`
+  values after refresh [GH-433]
+* `datasource/vcd_nsxv_dhcp_relay` crashes if no DHCP relay settings are present in Edge Gateway
+  [GH-446]
 * `resource/vcd_vapp_vm` `network` block changes caused MAC address changes in existing NICs
   [GH-436,GH-407]
 * Fix a potential data race in client connection caching when VCD_CACHE is enabled [GH-453]
 * `resource/vcd_vapp_vm` when customization.0.force=false crashes with interface {} is nil [GH-462]
```

Comment:
Would like to quote things like in below bullet points, not sure about the second quote being in the right place though:

```suggestion
* `resource/vcd_vapp_vm` when `customization.0.force=false` crashes with `interface {} is nil` [GH-462]
```

Reply:
Added on both